src/MemoryTracing/MemoryTracingUtilsTest.cpp

```diff
@@ -133,28 +95,43 @@ SwapCached:      0 kB)",
   const std::string kEmptyMeminfo = "";
 
   {
+    const SystemMemoryUsage kExpectedSystemMemoryUsage = [&] {
```

Comment:
I think for the test here, that is fine, but in general I would try to avoid capturing all locals (neither per reference nor per value) and explicitly list the used ones.
But do you need any capture here at all, that static values should be available anyways?

Reply:
Ah right! Thanks for pointing it out!